cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-azure-pipelines-overwrite-autodetected-variables/complete.json

```diff
@@ -18,5 +18,6 @@
     },
     "engine_requested": "OSS"
   },
-  "dependency_parser_errors": []
+  "dependency_parser_errors": [],
+  "task_id": "324625d6-d4c7-4e22-8e2b-ae9b3f33bf42"
```

Comment:
is this intended? Can we use a more obviously fake task_id if this is intended?

Reply:
yep, done